package.json

```diff
@@ -41,23 +41,23 @@
     "postinstall": "patch-package",
     "prebuild": "rimraf build/ build-bin/",
     "build": "npm run build:src && npm run catch-uncommitted",
-    "build:src": "npm run prettify && npm run lint && npm run build:automation && npm run build:fast && npm run build:doc",
+    "build:src": "npm run prettify && npm run lint && npm run build:fast && npm run build:test && npm run build:doc",
     "build:fast": "gulp build && tsc",
-    "build:automation": "tsc -P automation/tsconfig.json --noEmit",
-    "build:doc": "mkdirp doc/ && ts-node --transpile-only -P automation/tsconfig.json automation/capitanodoc/index.ts > doc/cli.markdown",
-    "build:standalone": "ts-node --transpile-only -P automation/tsconfig.json automation/run.ts build:standalone",
-    "build:installer": "ts-node --transpile-only -P automation/tsconfig.json automation/run.ts build:installer",
+    "build:test": "tsc -P ./tsconfig.dev.json --noEmit",
```

Comment:
In PR #1653 (older and still awaiting review), this change takes the form:
```
"pretest": "npm run build && tsc -P tests/tsconfig.json --noEmit"
```
I don't mind it one way or another, but we can't have both. :-)

Reply:
This one covers type-checking for both automation and tests in one go so should be faster in theory. It does include the main code as well that could in theory be removed for speed, but it's nice to have one config to check everything imo, and it inherits all the compiler options from the production tsconfig so it will help to keep that standardized :)

Reply:
I have now dropped the conflicting commit from PR #1653.